source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2447,16 +2473,46 @@ export class ImageSet extends CardElementContainer {
                 if (renderedImage) {
                     renderedImage.style.display = "inline-flex";
                     renderedImage.style.margin = "0px";
-                    renderedImage.style.marginRight = "10px";
+                    if (this.presentationStyle == Enums.ImageSetPresentationStyle.Default) {
+                        renderedImage.style.marginRight = "10px";
+                    }
 
                     Utils.appendChild(element, renderedImage);
                 }
             }
+            if (this.presentationStyle == Enums.ImageSetPresentationStyle.Stacked) {
+                this.applyStackedPresentationStyle();
+            }
         }
+        
 
         return element;
     }
 
+    applyStackedPresentationStyle() {
+        if (this._images[0].renderedImageElement) {
+            let dimension = StackedImageConfigurator.parseNumericPixelDimension(this._images[0].renderedImageElement.style.height);
+            let bgColor = this.getEffectiveBackgroundColor();
+            if (dimension && bgColor) {
+                let stackedImageConfigurator = new StackedImageConfigurator(this.pixelOffset, 
```

Comment:
Do we need any constraints on the offset?

I tried the sample with `-150`, and the images render outside of the card:

<img width="436" alt="image" src="https://user-images.githubusercontent.com/98650930/206778261-9cc69c16-9d98-4d89-a784-cff732ebbeb0.png">

Reply:
@anna-dingler; that's good point; going past each other as the images move closer doesn't make sense. since the user can achieve the same with the order of the images reversed. Added hard limit at the eclipse.